native/src/main/resources/ballerina-consolidate-packages-new.help

```diff
@@ -0,0 +1,23 @@
+NAME
+       ballerina-consolidate-packages-new - Create a consolidator package for the provided services
+
+
+SYNOPSIS
+       bal consolidate-packages new <comma-separated-list-of-services>
+
+
+DESCRIPTION
+       Creates a new package that consolidates the provided services into a single package.
+       This command optionally accepts a name for the consolidator package.
+
+
+OPTIONS
+       --name (Optional)
+           Package name for the consolidator. Default package name is 'consolidator'.
```

Comment:
@azinneera I am thinking whether `consolidator` is the right name for the package? As tool itself is the consolidator which consolidate into a package. Maybe `consolidated_package` or something like that could be the name

Reply:
Just wondering how Asgardeo org naming rule applies to a Ballerina package name?

Reply:
Made the name mandatory to be consistent with the `bal new` command.